lib/vmdb/fast_gettext_helper.rb

```diff
@@ -2,6 +2,11 @@ module Vmdb
   module FastGettextHelper
     def self.human_locale(locale)
       # TRANSLATORS: Provide locale name in native language (e.g. English, Deutsch or Português)
+
+      # We lazy load the message catalogs on first use so we need to find another
+      # way to get a human friendly locale besides loading all of the catalogs
+      # to get this "human friendly locale" value.
+      return locale
       human_locale = _("locale_name")
```

Comment:
We call this method on each locale we have a po file for... so, even if we have lazy loaded po files, this first call to `_("local_name")` loads the ENTIRE po file.  This takes 2.5+ seconds on the loading the 3 non-english po files.